packages/quick-edit-extension/src/cfs.ts

```diff
@@ -108,6 +115,63 @@ export class CFS
 	async seed(files: WorkerLoadedMessage["body"]) {
 		this.rootFolder = files.name ?? this.rootFolder;
 		this.createDirectory(Uri.parse(`cfs:/${this.rootFolder}/`));
+		this.writeFile(
```

Comment:
I don't know if we necessarily want that? I think people will want to be able to see the `jsconfig.json` that's being used to type-check their worker (even if only so they know what they're opting out of with `// @ts-no-check` if they want).

Reply:
`workers-types` I could see the argument for hiding, I guess. It feels like it serves a bit of a documentation purpose being visible though, as well as supporting click to definition in VSCode. I could be convinced otherwise though—not too set on that.

Reply:
VSCode adds a lock icon to the file, and shows a helpful message if you try to edit them: 
<img width="281" alt="Screenshot 2023-04-14 at 14 03 26" src="https://user-images.githubusercontent.com/28503158/232051339-a3c92f50-8ca4-4fef-adec-a5bd8459e9c8.png">
<img width="171" alt="Screenshot 2023-04-14 at 14 03 06" src="https://user-images.githubusercontent.com/28503158/232051352-8e3f76db-af2f-479c-9ae6-e0aeef1ec418.png">